src/Uno.Extensions.Core.Generators/Uno.Extensions.Core.Generators.csproj

```diff
@@ -0,0 +1,50 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<IsPackable>false</IsPackable>
+		<RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+		
+				<!--
+			Defines that this project output is a tool of the given package ID,
+			so nuget override will copy it in the 'tools' directory of that package instead of the 'lib' of the current package folder.
+		-->
+		<ToolOfPackage>Uno.Extensions.Reactive</ToolOfPackage>
```

Comment:
Is this still required? If so, it feels weird that this is Uno.Extensions.Core.Generators, instead of specific to Reactive

Reply:
Yes it's required, but not the right value ... changed it!